src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -260,6 +260,8 @@ public PlatformDetectorResult Detect(RepositoryContext context)
                 hasDevDependencies = true;
             }
 
+            var npmVersionSpec = packageJson?.engines?.npm?.Value as string;
```

Comment:
Should we add a log statement that Oryx has detected and will use this new version? Or are the logs explicit enough that the npm version in the package.json is being used?

Reply:
Per our discussion offline: we have an `echo` statement in the build script that mentions we're using the npm version spec in the `package.json` file right before we run the `npm install -g npm@<spec>` command.